### PR TITLE
Issue #6207: Added Xpath IT regression test for StaticVariableName

### PR DESCRIPTION
**Pull request by romani**
Issue #6207

from https://github.com/checkstyle/checkstyle/pull/13959

### CODE REVIEW

File: src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java
Comment (nrmancuso) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java:1]: Could we make this input a bit more interesting, perhaps declare a static field within a local class?

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java
Comment (romani) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java:1]: @aayushRedHat , can you create PR on this from account that Circle CI like?

Comment (aayushRedHat) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/staticvariablename/SuppressionXpathRegressionStaticVariableName2.java:1]: Done @romani

**Pull request closed without merging**